uk_bin_collection/uk_bin_collection/councils/RoyalBoroughofGreenwich.py

```
@@ -65,6 +66,29 @@ def parse_data(self, page: str, **kwargs) -> dict:
         offset_days = days_of_week.index(collection_day)
         week = collectionweek.index(week)
 
+        # Retrieve the list of dates amended by bank holidays such that we can adjust accordingly
+        holiday_URI = "https://www.royalgreenwich.gov.uk/recycling-and-rubbish/bins-and-collections/bank-holiday-collection-dates"
+        holiday_response = requests.get(holiday_URI, headers=headers)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)